robolectric/src/main/java/org/robolectric/plugins/DefaultSdkProvider.java

```diff
@@ -123,14 +123,19 @@ private DependencyJar getAndroidSdkDependency() {
         throw new UnsupportedClassVersionError(getUnsupportedMessage());
       }
 
-      String version =
-          String.join(
-              "-",
-              getAndroidVersion(),
-              "robolectric",
-              robolectricVersion,
-              "i" + PREINSTRUMENTED_VERSION);
-      return new DependencyJar("org.robolectric", "android-all-instrumented", version, null);
+      if (Boolean.parseBoolean(System.getProperty("robolectric.usePreinstrumentedJars", "true"))) {
```

Comment:
Will there exist a PR to update website for system properties(http://robolectric.org/configuring/)?

Reply:
It's in a different repo, so we'll need to do a follow up CL for that.

Reply:
Yep. It's better if we can update related system properties on doc/website after/along with the time we modify them.

Reply:
I have a CL to update the docs now here: https://github.com/robolectric/robolectric.github.io/pull/102

Reply:
Thanks for documentation updating.